reliquery/relic.py

```diff
@@ -247,6 +250,15 @@ def get_image(self, name: str) -> BytesIO:
         self.assert_valid_id(name)
         return self.storage.get_binary_obj([self.relic_type, self.name, "images", name])
 
+    def display_image(self, name: str) -> Image:
```

Comment:
IMO a better name for this is like get _pil_image or something liek that. You can use a pillow image without displaying it too!